development/ts-migration-dashboard/common/build-module-partitions.ts

```diff
@@ -126,15 +161,17 @@ function readFilesToConvert(): string[] {
  * Filters the given set of file paths according to the given allow list. As the
  * entry file paths could refer to TypeScript files, and the allow list is
  * guaranteed to be JavaScript files, the entry file paths are normalized to end
- * in `.js` before being filtered.
+ * in `.js` or `.jsx` before being filtered.
  *
  * @param filePaths - A set of file paths.
  * @param allowedFilePaths - A set of allowed file paths.
  * @returns The filtered file paths.
  */
 function filterFilePaths(filePaths: string[], allowedFilePaths: string[]) {
   return filePaths.filter((filePath) =>
-    allowedFilePaths.includes(filePath.replace(/\.tsx?$/u, '.js')),
+    allowedFilePaths.includes(
```

Comment:
It is possible that `.jsx` files were being mistakenly omitted from the dashboard or not being reported as being converted.